jwk.go

```diff
@@ -454,6 +455,8 @@ func (key rawJSONWebKey) ecPublicKey() (*ecdsa.PublicKey, error) {
 	switch key.Crv {
 	case "P-256":
 		curve = elliptic.P256()
+	case "secp256k1":
```

Comment:
Huh, is this definitely the right string? Maybe add a comment here and quote the appropriate RFC to make sure people reading this code in the future won't be confused about the dissonance.

Reply:
Yeah, I will add a reference to https://tools.ietf.org/html/draft-ietf-cose-webauthn-algorithms-03